src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp.js';
 import PropTypes from 'prop-types';
 
 const ChatEntry = (props) => {
+  const timeEntry=props.timeStamp
```

Comment:
Nitpick - there should be whitespace around the equal sign. When we pass props to a component, then by convention we should omit the spaces around the equal sign, but when we're declaring variables we should have them.